src/entries/main.jsx

```diff
@@ -1,5 +1,5 @@
-import HelloWorld from 'components/HelloWorld';
+import EventForm from 'components/EventForm';
 import React from 'react';
 import { render } from 'react-dom';
 
-render(<HelloWorld/>, document.getElementById('hello-world'));
```

Comment:
I think you shouldn't have the `EventForm` be the "root". The root should be some kind of higher `App`-like component.